ocaml/sdk-gen/go/gen_go_binding.ml

```diff
@@ -44,20 +44,84 @@ let render_api_messages_and_errors destdir =
   generate_file ~rendered:messages_rendered ~destdir
     ~output_file:"api_messages.go"
 
+let render_convert_header () =
+  let obj =
+    `O
+      [
+        ("name", `String "convert")
+      ; ( "modules"
+        , `O
+            [
+              ("import", `Bool true)
+            ; ( "items"
+              , `A
+                  [
+                    `O [("name", `String "fmt"); ("sname", `Null)]
+                  ; `O [("name", `String "math"); ("sname", `Null)]
+                  ; `O [("name", `String "reflect"); ("sname", `Null)]
+                  ; `O [("name", `String "strconv"); ("sname", `Null)]
+                  ; `O [("name", `String "time"); ("sname", `Null)]
+                  ]
+              )
+            ]
+        )
+      ]
+  in
+  render_template "FileHeader.mustache" obj ~newline:true ()
+
+let render_converts converts destdir =
+  let templates =
+    [
+      "ConvertSimpleType.mustache"
```

Comment:
Would any one of this list require all of the following which are coded in gen_go_helper.ml?
```
      [
        ("simple_type", simple_types)
      ; ("int", of_int)
      ; ("float", of_float)
      ; ("time", time)
      ; ("ref", ref_types)
      ; ("set", set_types)
      ; ("record", records)
      ; ("interface", interface)
      ; ("map", maps)
      ; ("enum", enums)
      ; ("batch", event_batch)
      ; ("option", options)
      ]
```
If not, each of them could be coded together.

Reply:
These related code has been split to the following PR which will raised after this one got merged.